script.js

```diff
@@ -0,0 +1,53 @@
+const bookForm = document.querySelector('.book-form');
+const bookContainer = document.querySelector('.book-container');
+const titleInput = bookForm.title;
+const authorInput = bookForm.author;
+
+const books = JSON.parse(localStorage.getItem('books')) || [];
+
+const addBook = (title, author) => {
+  books.push({
+    title,
+    author,
+  });
+  localStorage.setItem('books', JSON.stringify(books));
+  return { title, author };
+};
+
+const createBook = ({
+  title, author,
+}, index) => {
+  const bookDiv = document.createElement('div');
+  const bookTitle = document.createElement('h2');
+  const bookAuthor = document.createElement('p');
+  const AremoveBtn = document.createElement('button');
+  const Ahr = document.createElement('hr');
+
+  bookTitle.innerText = `${title}`;
+  bookAuthor.innerText = `${author}`;
+  AremoveBtn.innerText = 'Remove';
+
+  bookDiv.append(bookTitle, bookAuthor, AremoveBtn, Ahr);
+  bookContainer.appendChild(bookDiv);
+
+  AremoveBtn.addEventListener('click', (e) => {
+    e.preventDefault();
+    const remaining = books.filter((book, bookIndex) => bookIndex !== index);
+    localStorage.setItem('books', JSON.stringify(remaining));
+    bookDiv.remove();
+  });
```

Comment:
- [ ]  When I click the remove button to remove a certain book, the book is not removed from the localStorage. Please make sure the local storage is updated after removing a specific book. You may use the dev tools to check if the right book is removed from the local storage.

In the screenshot below, the two books I deleted are still in the local storage

<img width="766" alt="image" src="https://user-images.githubusercontent.com/77190203/176051883-ca6083db-a10c-4903-b7d0-74d4582dab62.png">